src/main/features/response/paths.ts

```diff
@@ -28,6 +28,8 @@ export class Paths {
   static readonly defendantHowMuchOwed = new RoutablePath(`${responsePath}/how-much-owed`)
   static readonly defenceRejectAllOfClaimPage = new RoutablePath(`${responsePath}/reject-all-of-claim`)
   static readonly defenceRejectPartOfClaimPage = new RoutablePath(`${responsePath}/reject-part-of-claim`)
+  static readonly defenceFullPartialPaymentOptionsPage = new RoutablePath(`${responsePath}/when-will-you-pay`)
```

Comment:
I would remove `FullPartial` from the name as it does not add much value

Reply:
Done

Reply:
Partially done, out of two added pages only one was changed 👎